cli/tests/unit/net_test.ts

```diff
@@ -240,7 +240,9 @@ unitTest(
     assertEquals(bob.addr.hostname, "127.0.0.1");
 
     const sent = new Uint8Array([1, 2, 3]);
-    await alice.send(sent, bob.addr);
+    const byteLength = await alice.send(sent, bob.addr);
+
+    assertEquals(byteLength, 3);
```

Comment:
I'm surprised this test works. 

please update the type declaration of DatagramConn.send in `cli/js/lib.deno.unstable.d.ts` too